enhancements/etcd/automated-backups-no-config.md

```diff
@@ -0,0 +1,256 @@
+* Backups are saved to a default location on master's node disk using `hostPath` volume.
+* Backups are taken according to a default schedule.
+* Backups are taken according to a default retention policy.
+
+### Non-Goals
+
+* Save cluster backups to remote cloud storage (e.g. S3 Bucket).
+  - This could be a future enhancement or extension to the API.
+  - Having backups in an independent storage from OCP cluster is crucial for Disaster Recovery scenarios.
+* Automate cluster restoration.
+* Provide automated backups for non-self hosted architectures like Hypershift.
+
+## Proposal
+
+To enable automated etcd backups of an Openshift cluster, a default backup option is to be taking sing a `SideCar` container within each etcd pod, with no configuration from user.
+
+### User Stories
+
+* As a cluster administrator I would like OCP cluster backup to be taken without configuration.
+* As a cluster administrator I would like to schedule recurring OCP cluster backups so that I have a recent cluster state to recover from in the event of quorum loss (i.e. losing a majority of control-plane nodes).
+
+### API Extensions
+
+No [API](https://github.com/openshift/enhancements/blob/master/enhancements/etcd/automated-backups.md#api-extensions) changes are required, since this approach work independently with default config.
+
+### Topology Considerations
+TBD
+#### Hypershift / Hosted Control Planes
+TBD
+#### Standalone Clusters
+TBD
+#### Single-node Deployments or MicroShift
+TBD
+### Workflow Description
+TBD
+
+## Design Details
+
+A `SideCar` container is being added within each etcd pod in Openshift cluster in order to take backups without any provided configuration from user's perspective.
+
+As the `SideCar` is being deployed alongside each etcd cluster member, it is possible to keep backups across all master nodes.
+
+Since this work will be enabled with no configuration, it is possible to use define a default values for the `Schedule`, `Retention` independently.
+
+This work utilizes `hostPath` approach, since the `SideCar` container has access to the etcd pod's file system.
+
+Before deciding upon this approach, several approaches that have been investigated. See [Alternatives](#Alternatives).
+
+### Open Questions
+
+* How to distinguish between `NoConfig` backups and configs that are triggered using `EtcdBackup` CR.
+  * The `NoConfig` backups relies on `EtcdBackup` CR with `default` name.
+  * The cluster-etcd-operator reacts to the `default` CR by deploying the backup sidecar containers alongside each etcd member. 
+
+
+* Upon enabling the `AutomatedBackup` feature gate, which approach should be used and according to what criteria.
+  * As the `Noconfig` backups is orthogonal to the [automated backup of etcd](https://docs.openshift.com/container-platform/4.15/backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.html#creating-automated-etcd-backups_backup-etcd), it has been decided to use the same feature gate.
```

Comment:
That means you have to get both features ready and tested before either can be promoted. Strongly advise to use a separate gate

Reply:
it is one feature gate, that has been used in https://github.com/openshift/enhancements/blob/master/enhancements/etcd/automated-backups.md 

The `PeriodicBackupController` is feature gated with it https://github.com/openshift/cluster-etcd-operator/blob/ddbab6c9a4e65d3dbcd5a312caabd5e05879ef95/pkg/operator/periodicbackupcontroller/periodicbackupcontroller.go#L111

Reply:
Are you and your PM/lead willing to promote the periodic backups only once this feature has also passed full QE testing?

Reply:
I am not the one who answers this question to be honest :D :D

Reply:
Who is?

Reply:
the PM or Lead :D :D

Reply:
And who are they? 😂

Reply:
@hasbro17 @tjungblu @racedo